static/js/place_overview.jsx

```diff
@@ -336,20 +347,27 @@ Overview.propTypes = {
 
 class ChildPlace extends Component {
   render() {
-    return (
-      <React.Fragment>
-        {Object.keys(this.props.childPlaces).map((placeType) => (
-          <div key={placeType}>
-            <div>{placeType}</div>
-            {this.props.childPlaces[placeType].map((place) => (
-              <a key={place["dcid"]} href={"/place?dcid=" + place["dcid"]}>
-                {place["name"]}
-              </a>
-            ))}
-          </div>
-        ))}
-      </React.Fragment>
-    );
+    if (Object.keys(this.props.childPlaces).length > 0) {
+      return (
+        <React.Fragment>
+          <div id="child-place-title">Child Places</div>
+          {Object.keys(this.props.childPlaces).map((placeType) => (
+            <div key={placeType}>
+              <div className="child-place-type">{placeType}</div>
+              {this.props.childPlaces[placeType].map((place) => (
+                <a
+                  key={place["dcid"]}
+                  className="child-place-link"
+                  href={"/place?dcid=" + place["dcid"]}
+                >
+                  {place["name"]},
```

Comment:
Is there a way to leave the comma out if it's the last one?

Reply:
add a check for last item